apps/src/lab2/progress/ProgressManager.ts

```diff
@@ -6,6 +6,11 @@ import {Condition, Validation} from '@cdo/apps/lab2/types';
 // Abstract class that validates a set of conditions. How
 // the validation works is up to the implementor.
 export abstract class Validator {
+  abstract shouldValidateWithExemplar(): boolean;
```

Comment:
Optional: the `ProgressManager` gets handed the level's `validations` in the `ProgressContainer` component, so we could simply get the `exemplarSettings` from the level there as well, instead of having the lab implementation pass it back up.

Reply:
I've attempted to implement this. In order for the `Validator` to get the `exemplarSettings`, I created a setter in the `ProgressManager` that is called when we initially set the validator instance.